aicssegmentation/tests/test_WorkflowStep.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from aicssegmentation.structure_wrapper.WorkflowEngine import WorkflowStep
+from skimage import data
+import pytest
+
+
+class TestWorkflowEngine:
```

Comment:
Bad copy paste - this should be `TestWorkflowStep`. Not sure if pytest is even executing this right now since there are 2 test classes with the same name.